### PR TITLE
Create pt-br.json

### PR DESCRIPTION
**Pull request by eduaddad**
translation into brazilian portuguese

### CODE REVIEW

File: i18n/pt-br [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+	"@metadata": {
+		"authors": [
+			"Reception123"
+		]
+	},
+	"analytics": "MatomoAnalytics",
+	"matomoanalytics-desc": "Integra análises de Matomo e MediaWiki",
+	"matomoanalytics-header": "Você pode usar o formulário abaixo para selecionar diferentes estatísticas para ver que armazenamos usando o Matomo..",
+	"matomoanalytics-form-stat1": "Estatística 1",
+	"matomoanalytics-form-stat2": "Estatística 2"
```
Comment (paladox) [inline i18n/pt-br]: needs to have "," at the end please.

File: i18n/pt-br [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+	"@metadata": {
+		"authors": [
+			"Reception123"
```
Comment (paladox) [inline i18n/pt-br]: Change the user here to you, so your attributed correctly (ie so your given credit)

File: i18n/pt-br (hunk 1 above)
Comment (eduaddad) [reply i18n/pt-br]: outdated?

I used the English version and did the translation

File: i18n/pt-br (hunk 2 above)
Comment (eduaddad) [reply i18n/pt-br]: done

File: i18n/pt-br.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+	"@metadata": {
+		"authors": [
+			"Eduaddad"
+		]
+	},
+	"analytics": "MatomoAnalytics",
+	"matomoanalytics-desc": "Integra análises de Matomo e MediaWiki",
+	"matomoanalytics-header": "Você pode usar o formulário abaixo para selecionar diferentes estatísticas para ver que armazenamos usando o Matomo..",
+	"matomoanalytics-form-stat1": "Estatística 1",
+	"matomoanalytics-form-stat2": "Estatística 2"
```
Comment (MacFan4000) [inline i18n/pt-br.json]: Comma at the end of this line please

Comment (eduaddad) [reply i18n/pt-br.json]: done

**Pull request merged**